app/assets/stylesheets/application.scss

```
@@ -0,0 +1,63 @@
+body {
+  padding-top: 70px;
+}
+
+.header_logo {
+  width: 221px;
+  margin-left:20px;
+}
+
+.user_icon {
+  width:40px;
+  height:40px;
```

```suggestion
  width:　40px;
  height:　40px;
```
半角スペース入れましょう！
他にも同様の修正箇所があるのでセルフチェックしてみてください！